src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/FSharp/FSharpCommandLineParserService.cs

```diff
@@ -22,7 +21,7 @@ internal class FSharpCommandLineParserService : ICommandLineParserService
         private const string LongReferencePrefix = "--reference:";
 
         [ImportMany]
-        private readonly IEnumerable<Action<string, ImmutableArray<CommandLineSourceFile>, ImmutableArray<CommandLineReference>, ImmutableArray<string>>> _handlers = null;
+        private readonly IEnumerable<Action<string, ImmutableArray<CommandLineSourceFile>, ImmutableArray<CommandLineReference>, ImmutableArray<string>>> _handlers = Array.Empty<Action<string, ImmutableArray<CommandLineSourceFile>, ImmutableArray<CommandLineReference>, ImmutableArray<string>>>();
```

Comment:
I would prefer these are not filled in as it would be a bug if they are null by the time they are touched.

Reply:
Good point. The alternative is to attribute them with `[NotNull, AllowNull]` and leave off the `?` modifier. I'll do that before merging.

Reply:
I can't actually get this to work with attributes alone. Leaving the type non-nullable and assigning `null!` works. I've made the change.

An observation while searching for instances of this: most `ImportMany` fields use `OrderPrecedenceImportCollection` which is always initialised and would hide any import bugs. I haven't checked but suspect MEF would assign an empty enumerable here anyway.